msteams-platform/bots/bot-concepts.md

```diff
@@ -572,6 +572,9 @@ An extensive dialog between your bot and the user is a slow and complex way to g
 * **Optimize size of knowledgebase for quicker interaction**
   One of the disadvantages of bots is that it's difficult to maintain a large retrieval knowledge base with unranked responses. Bots are best suited for short, quick interactions, and not sifting through long lists looking for an answer.
 
+> [!NOTE]
+> Teams platform supports Transport Layer Security (TLS) versions 1.0, 1.1, and 1.2, but not TLS version 1.3. Ensure you configure your bot environment accordingly.
```

Comment:
I'm not sure whether TLS1.0 and 1.1 should be included as they are legacy protocols with security risks. I'm confirming this with SMBA team and will get back to you.